projects/js-packages/charts/src/components/shared/utils.ts

```diff
@@ -0,0 +1,69 @@
+import { getStringWidth } from '@visx/text';
+import { format } from 'd3-format';
```

Comment:
I wonder if we would be able to leverage `@automattic/number-formatters` instead?

Reply:
That's a good idea. I didn't know we have that package. I'll check it out.

Reply:
I've removed `d3-format` dependency and use `@automattic/number-formatters` instead. d705d4098d3cd15c4d95024cb459554284241ab5